addons/canal/canal_v3.20.yaml

```diff
@@ -3791,7 +3791,7 @@ spec:
         # This container installs the CNI binaries
         # and CNI network config file on each node.
         - name: install-cni
-          image: '{{ Registry "docker.io" }}/calico/cni:v3.20.2'
+          image: '{{ Registry "quay.io" }}/calico/cni:v3.20.2'
```

Comment:
The `3.20.2` version is missing multi-arch manifest (so would not work on ARM nodes). We can:
 - a) leave `3.20.2` on dockerhub
 - b) bump the version to `3.20.5` 
 
![image](https://user-images.githubusercontent.com/15926980/178431714-3b49f946-5a5b-43e7-94eb-399de08de7ac.png)

Reply:
bumped to `3.20.5`